SudokuLib/Board.cpp

```diff
@@ -4,8 +4,10 @@ using std::copy;
 
 Board::Board()
 {
-	for (board_index_datatype row = 0; row < 9; row++) {
-		for (board_index_datatype column = 0; column < 9; column++) {
+	for (board_index_datatype row = 0; row < 9; row++)
+	{
+		for (board_index_datatype column = 0; column < 9; column++)
+		{
```

Comment:
Just a comment, it may be a good idea to isolate formatting changes in their own pr.
Btw, I'll add a .clang-format to the project later today, so we can all easily auto-format the project.
(my current timezone is GMT+2)